### ISSUE TITLE
avocode beta-testing from no reason  no error jump

### ISSUE THREAD

**Issue by vAhyThe** [avocode/avocode#922]
I cant see any layers but avocode still can take css properties

![image](https://cloud.githubusercontent.com/assets/6084353/16078227/fbfcb278-32fd-11e6-9cf3-4cbe6921fde1.png)

**Comment by vojtatranta** [comment]
Hi, what OS do you use?

**Comment by vAhyThe** [comment]
MacBookPro , OS X 10.11.5 (15F34)

**Comment by jakubzitny** [comment]
Thanks for replying @vAhyThe. This is a problem with Avocode that happens very very rarely and we haven't been able to replicate it and fix it. We've seen it though. Can you confirm that it happened only this one time for you? Can you use the app normally?

**Comment by vAhyThe** [comment]
@jakubzitny Yes it was only once and now work perfectly. if it will happen again I'll let you know.

**Comment by vAhyThe** [comment]
@jakubzitny @vojtatranta  its happen again. You can try to simulate. When you click `command` + `h` and after that click to the icon in dockbar.

**Comment by petrbrzek** [comment]
@vAhyThe Is this still happening with 2.8.4?

**Comment by vAhyThe** [comment]
@petrbrzek  its look ok  for now. If something change i will create new issue. 
btw Great work guys !!

**Comment by connor-bean** [comment]
Awesome! Thanks @vAhyThe!

We will close this out. Please reopen if you see this issue again or open a new ticket.

Thanks!
Connor, Avocode Support
